octavia_proxy/tests/unit/api/drivers/elbv3/test_driver.py

```diff
@@ -81,6 +151,37 @@ def test_loadbalancers_qp(self):
             a='b'
         )
 
+    def test_loadbalancer_get(self):
+        self.driver.loadbalancer_get(self.sess, 'test', self.lb)
+        self.sess.vlb.find_load_balancer.assert_called_with(
+            name_or_id=self.lb, ignore_missing=True)
+
+    def test_loadbalancer_create(self):
+        self.driver.loadbalancer_create(self.sess, self.lb)
+        self.sess.vlb.create_load_balancer.assert_called_with(
+            **self.fakeCallCreate)
+
+    def test_loadbalancer_update(self):
+        attrs = {
+            'description': 'New Description',
+            'operating_status': 'ACTIVE',
+        }
+        self.driver.loadbalancer_update(self.sess, self.lb, attrs)
+        self.sess.vlb.update_load_balancer.assert_called_with(
+            self.lb.id, **attrs)
+
+    def test_loadbalancer_delete(self):
+        self.driver.loadbalancer_delete(self.sess, self.lb)
+        self.sess.vlb.delete_load_balancer.assert_called_with(self.lb.id)
+
+    def test_loadbalancer_delete_cascade(self):
```

Comment:
later:
I am kinda blind - where you mock health_monitors?

BTW, there is a nice API function: https://docs.otc.t-systems.com/api/elb/elb_zq_fz_0004.html The same should be also existing for the elbv3. We could use it exactly as a way to gather all dependent resources of the LB so that 1 API call should be enough

Reply:
now using get_load_balancer_statuses for elbv3, left load_balancer_status_tree if cascade feature will be removed from elbv2 (or until update for get_load_balancer_statuses in sdk)

Reply:
great. cascaded_delete IS already removed from v2 since quite some time, we just didn't knew about that.